tldw_Server_API/tests/Chatbooks/test_chatbooks_sync_contracts.py

```
@@ -0,0 +1,264 @@
+        "success": True,
+        "message": "Import completed",
+        "job_id": None,
+        "imported_items": {"note": 2},
+        "warnings": ["from-wrapper"],
+    }
+    assert captured["import_media"] is False
+    assert captured["import_embeddings"] is False
+
+
+@pytest.mark.unit
+def test_sync_import_service_counts_renamed_items_as_imported(tmp_path, monkeypatch):
```

Ruff (0.15.9)

[warning] Unused function argument: `tmp_path`

(ARG001)